src/modules/profile/utils.ts

```diff
@@ -0,0 +1,23 @@
+import { Profile } from '../../modules/profile/types'
+
+export const lambdaProfileToContentProfile = (profile: Profile) => {
+  return {
+    ...profile,
+    avatars: profile.avatars.map(avatar => ({
+      ...avatar,
+      avatar: {
+        ...avatar.avatar,
+        snapshots: {
+          body: avatar.avatar.snapshots.body.replace(
```

Comment:
This is just a question, but, have we confirmed with platform that those are going to be the only snapshots that will be here? I'm a bit worried that there are more or that they add more of them in the future, making this code break the profiles. Maybe we can add a function here that processes all snapshots and performs the replace.

Reply:
Yeah, I think I could iterate over the whole object to cover the case they add more data to the snapshots object! will update soon

Reply:
added!